### PR TITLE
Release notes update (map view)

### PR DESCRIPTION
**Pull request by phweyland**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -85,6 +85,14 @@ making a backup is strongly advised.
   with different default settings, because input color profiles and 
   demosaicing need at least a rough white balancing early in the pipe.
 
+  Map view has evolved. Images close to each other are now grouped and a count of
```
Comment (TurboGit) [inline RELEASE_NOTES.md:88]: other -> others

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -85,6 +85,14 @@ making a backup is strongly advised.
...
+  grouped images is displayed on the bottom-left corner. This provides better performances
+  when the map covers thousands of images. Scrolling over a thumb scrolls through
```
Comment (TurboGit) [inline RELEASE_NOTES.md:90]: thumb -> grouped thumbs

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -85,6 +85,14 @@ making a backup is strongly advised.
...
+  the images of the group. Dragging of images has also been improved.
```
Comment (TurboGit) [inline RELEASE_NOTES.md:91]: Maybe talk here about the shift drag for a group?

**Review by TurboGit — COMMENTED** [review]
Some comments.

File: RELEASE_NOTES.md (hunk 1 above)
Comment (phweyland) [reply RELEASE_NOTES.md:88]: I've copied that from elstoc's text in dt doc, should be correct, shouldn't it  ?

File: RELEASE_NOTES.md (hunk 2 above)
Comment (phweyland) [reply RELEASE_NOTES.md:90]: I'll do.

File: RELEASE_NOTES.md (hunk 3 above)
Comment (phweyland) [reply RELEASE_NOTES.md]: ok

File: RELEASE_NOTES.md (hunk 1 above)
Comment (parafin) [reply RELEASE_NOTES.md:88]: other is correct

Comment (TurboGit) [reply RELEASE_NOTES.md:88]: If elstoc wrote that it is correct no question about that! So I'm worng.

**Comment by phweyland** [comment]
Changes requested done. Added count number color to indicate if images are at the same place or not.

**Review by TurboGit — APPROVED** [review]
Thanks !

**Pull request merged**